src/async-polling.js

```diff
@@ -1,79 +1,66 @@
-(function () {
-    'use strict';
+const EventEmitter = require('events');
```

Comment:
Doesn't this break web browser support?

Reply:
Perhaps add a polyfill for older browsers?

Reply:
I don't think the classes should be a problem! add babel to grunt 😃